### ISSUE TITLE
ROC-AUC score on Tox21 lower than reported in Tutorial 4 

### ISSUE THREAD

**Issue by Fairly** [deepchem/deepchem#2346]
## ❓ Questions & Help

Hi there,
first, thank you for maintaining this awesome project!

I'm going through the tutorials and find that the 'roc_auc' in the tutorial is about 0.77. While I've tried to run the code many times without any changes and the roc_auc is never higher than 0.7. Does anyone face the same problem with me? Is there anything I can do to find the reason?

Thanks!

**The result in Tutorial 4:**
training set score: {'roc_auc_score': 0.9550063590563469}
test set score: {'roc_auc_score': **0.7781819573695475**}

**My run with colab:**
training set score: {'roc_auc_score': 0.9589362210016299}
test set score: {'roc_auc_score': **0.6836742718937837**}

**Comment by peastman** [comment]
We recently completed an overhaul of the MoleculeNet loader functions.  As part of that, we changed the default splitting method for a lot of them.  Previously most of them used random splitting by default, which often overestimates how well it will work on independent data.  We've now switched most of them to use scaffold splitting by default.  That's probably the reason for the change in test set performance.

**Comment by peastman** [comment]
Keep in mind that a higher test set score isn't necessarily better.  Probably it's the opposite: the scaffold splitter gives a better (that is, more realistic) picture of how well the model would work on truly unrelated data.
